bzl/deps.bzl

```diff
@@ -0,0 +1,78 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def djinni_deps():
+
+    native.android_sdk_repository(name = "androidsdk")
```

Comment:
yea these android and swift/apple/kotlin examples probably should probably have their own WORKSPACES instead of being part of the main workspace, otherwise you're polluting with extra deps for no reason @LiFengSC

Reply:
Yes.  I didn't know we can have nested WORKSPACEs

Reply:
you cannot, but typically example workspaces are `.bazelignore`'ed and set up to be standalone and inner workspaces are made to work with `local_repository` and relative paths to the top level one:

https://github.com/bazelbuild/rules_kotlin/blob/master/.bazelignore
https://github.com/bazelbuild/rules_kotlin/blob/master/examples/android/WORKSPACE#L5

Reply:
I'll move these back to the top-level WORKSPACE for now